app/src/main/java/ru/uniquenature/myapplication/FragmentMoviesDetails.kt

```diff
@@ -12,14 +12,9 @@ import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.bumptech.glide.Glide
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.coroutineScope
-import kotlinx.coroutines.launch
 import ru.uniquenature.myapplication.data.Movie
-import ru.uniquenature.myapplication.data.loadMovies
 
-class FragmentMoviesDetails(private val position: Int) : Fragment() {
+class FragmentMoviesDetails(private val movies: List<Movie>, private val position:Int) : Fragment() {
```

Comment:
@sanya-arsenal Very bad solution, do not pass anything through Fragment constructor. Recreation of Fragment will call default one without any data. You need to pass only one movie and using Fragment arguments